core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -990,6 +979,9 @@ public void sleepTimerUpdate(SleepTimerUpdatedEvent event) {
         if (event.isOver()) {
             mediaPlayer.pause(true, true);
             mediaPlayer.setVolume(1.0f, 1.0f);
+            int newPosition = mediaPlayer.getPosition() - (int) SHORT_REWIND;
```

Comment:
Shouldn't we just rewind `PlaybackServiceTaskManager.SleepTimer.NOTIFICATION_THRESHOLD`, which is the time that the sleep timer uses for fading out?

Reply:
Personally I don't have issues to hear the first half of `NOTIFICATION_THRESHOLD`, but only the last few seconds. It may be a too large rewind. Maybe use `NOTIFICATION_THRESHOLD / 2`?

Reply:
Sounds good 👍